app/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ plugins {
 android {
     namespace = "com.sebastianvm.mango"
     compileSdk = 33
+    compileSdkPreview = "UpsideDownCakePrivacySandbox"
```

Comment:
I was getting build errors without this.

Reply:
Ah, feel free to undo my changes! If you decide to, I would restack on main first